shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/domain/interactor/StepQuizInteractor.kt

```diff
@@ -84,5 +84,7 @@ class StepQuizInteractor(
     }
 
     suspend fun generateGptCodeWithErrors(stepId: Long): Result<String> =
-        submissionsRepository.generateCodeWithErrors(stepId)
+        submissionsRepository.generateCodeWithErrors(stepId).map { code ->
+            code.lineSequence().filter { !it.startsWith("```") }.joinToString("\n")
```

Comment:
OK, but I think it is possible to modify the system prompt in a way that will remove any markdown code block delimiters from the generated code

Reply:
Agree, with this solution I would like to make sure that in any situation prompt will work correctly. We already have instruction for markdown. Testing new prompt is quite hard because every new deployment to backend takes about 40 minutes.